Documentation/project-docs/developer-guide.md

```diff
@@ -96,67 +112,61 @@ build-managed -debug -buildArch=x64
 build-managed -debug -buildArch=x64 -os=Linux
 ```
 
-- Building only the managed libraries with /flp:v=diag. It doesn't restore packages, build tests or build packages.
+##Build Tests
+To build the tests and run them you can call build-test script. The same parameters you pass to build should also be passed to build-tests script to ensure you are building and running the tests on the same configuration you have build the product on.
+
+**Examples**
+- The following shows how to build tests but not run them
 ```
-build-managed -binaries -verbose
+build-tests -skiptests
 ```
 
-- Building the managed libraries and the tests but not running tests.
+- The following builds and runs all tests for netcoreapp.
 ```
-build-managed -skiptests
+build-tests -debug -framework netcoreapp
```

Comment:
Please change the example to something that is not default (e.g. release and/or netfx)